OpenRA.Mods.RA2/UtilityCommands/ImportRA2MapCommand.cs

```diff
@@ -443,10 +444,12 @@ static void ReadTiles(Map map, IniFile file, int2 fullSize)
 				var rx = mf.ReadUInt16();
 				var ry = mf.ReadUInt16();
 				var tilenum = mf.ReadUInt16();
-				/*var zero1 = */mf.ReadInt16();
+				/*var zero1 = */
+				mf.ReadInt16();
```

Comment:
```suggestion
				mf.ReadInt16(); // zero1
```

Reply:
Please ignore the map importer, I have a follow-up PR for it ready.
If you wish I can add it here, it's just 1 commit 🤷‍♂️